src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -14,7 +14,7 @@ const Body = BaseForm.extend({
     // SIW customization hook for registration
     this.settings.preRegistrationSubmit(this.model.toJSON(),
```

Comment:
In V1, we sends `this.model.attributes` to this callback
https://github.com/okta/okta-signin-widget/blob/master/src/RegistrationController.js#L149-L150

Reply:
> In V1, we sends `this.model.attributes` to this callback
> https://github.com/okta/okta-signin-widget/blob/master/src/RegistrationController.js#L149-L150

@haishengwu-okta we had this conversation https://github.com/okta/okta-signin-widget/pull/1747#discussion_r594507570